src/_includes/newsletter-sign-up.html

```diff
@@ -18,25 +18,6 @@
 <!-- Begin Constant Contact Active Forms -->
 <script>
   var _ctct_m = "e240cb4001c90b8b1e2829cb8b017263";
```

Comment:
I think a quick comment saying this is an identifier for the form (I'm guessing?) or something